infra/buckets/gs_buckets.tf

```diff
@@ -4,7 +4,7 @@ resource google_storage_bucket dss_gs_bucket {
   provider = "google"
   location = "${length(var.DSS_GS_BUCKET_REGION) > 0 ?
     "${var.DSS_GS_BUCKET_REGION}" : "${var.GCP_DEFAULT_REGION}"}"
-  storage_class = "REGIONAL"
+  storage_class = "${var.DSS_DEPLOYMENT_STAGE == "integration" ? "STANDARD" : var.DSS_DEPLOYMENT_STAGE == "staging" ? "STANDARD" : "REGIONAL"}"
```

Comment:
`dev` is regional?

Reply:
Sorry for the convoluted logic, will fix.